strawberryfields/ops.py

```diff
@@ -1666,15 +1682,14 @@ class GraphEmbed(Decomposition):
 
     Args:
         A (array): an :math:`N\times N` complex or real symmetric matrix
-        max_mean_photon (float): threshold value. It guarantees that the mode with
-            the largest squeezing has ``max_mean_photon`` as the mean photon number
-            i.e., :math:`sinh(r_{max})^2 ==` max_mean_photon
+        mean_photon (float): mean_photon (float): It guarantees that the mean photon number in the pure Gaussian state
+            representing the graph satisfies  :math:`\sum_i sinh(r_{i})^2 ==` ``mean_photon``
         make_traceless (boolean): removes the trace of the input matrix
```

Comment:
do we want to add the additional comment here that also appears in `graph_embed`?